src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommandProxy.cs

```diff
@@ -89,55 +89,55 @@ internal bool HasHostWindow
         {
             get
             {
-                return (bool)_graphicalHostReflectionWrapper.GetPropertyValue("HasHostWindow");
+                return (bool)_graphicalHostReflectionWrapper.GetPropertyValue(nameof(HasHostWindow));
```

Comment:
The same. HasHostWindow is a property of GraphicalHostReflectionWrapper.

I see the same patterns below.